spotbugs/etc/messages.xml

```diff
@@ -1811,6 +1811,23 @@ factory pattern to create these objects.</p>
       ]]>
     </Details>
   </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.FindHidingSubClass">
+    <Details>
+      <![CDATA[
+            <p>
+              Hiding happens when a sub class defines a static method
+              with same header (signature plus return type) as any of its super classes.
+              In case of method hiding, the invoked method is determined based on the specific qualified name or
+              method invocation expression used at the calling site.
+              The results are often unexpected, although the Java language provides unambiguous rules method invocation for method hiding.
```

Comment:
There's a missing word here. "provides unambiguous rules [for? regarding?] method invocation"

Reply:
Line 9020 is the same, so if you change it here, don't forget to update that as well.

Reply:
addressed using 'for'